api_root.md

```diff
@@ -44,14 +44,12 @@ In this project it is automatically generated a set of GraphQL queries and mutat
 <br/><br/>
 ### Batch Data Exporting
 
-Unfortunately current NodeJS GraphQL implementation used in Zendro does not support batch download in a fully optimal way because of lack of the non-blocking response data streaming (see [this](https://github.com/graphql/graphql-js/issues/1537) discussion). When it is required to join selected fields of the related data models and get it as a separate file stream, using the *export* service would be the correct solution.
+It is possbile to download all records for one model by batches. Meanwhile, records would be exported as a CSV file, in which each field is quoted via `"` for reducing the ambiguity among fields.
```

Comment:
It is possbile to download all records for one model in batches. All records will be exported as a CSV file, in which each field is quoted via `"` to reduce the ambiguity among fields.